### PR TITLE
Add minutes for 05/09/2018 core team meeting 

### PR DESCRIPTION
**Pull request by BKiddo**

### CODE REVIEW

File: 2018/2018-09-05.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	2. Alistair has only just got access to the previous budget. After we draft a budget we should arrange a meeting with the trustees to discuss fundraising. There may be a possible lead from ASV. We should begin to draft a document for the trustee meeting.
+3.
+	1. Timing/van details need to be agreed for the sorting session this weekend, and information sent out. Saffron could have value but we will decide what functionality we want in the next few weeks, this will depend on Jake's survey.
+	2. Andy BS is open to mentoring ideas. PiWars get teams to physically mentor each other which may be an interesting option.
+4.
+	1. There was a lead for a custom PCB manufacturer which is worth following up.
+	2. Not a core team decision.
+	3.
```
Comment (RealOrangeOne) [inline 2018/2018-09-05.md]: nitpick: Looks like you've got an empty number here

Comment (BKiddo) [reply 2018/2018-09-05.md]: I left it empty because it was in the agenda but (to my knowledge) wasn't discussed in the meeting.

Comment (PeterJCLaw) [reply 2018/2018-09-05.md]: Interesting. Maybe the notes should be explicit about this -- perhaps put `(not discussed)`?

File: 2018/2018-09-05.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	5. Not as many teams as expected. Most teams are in the south. We need to say we are in the UK as many teams are from remote countries. 
+	6. There may be a GDPR git exception, we might be able to keep gitlogs but not git repo data. 
+	7. Depending on the events of the weekend we may have a quick video call on Sunday.
+
+## Action Points:
+
+### General:
+The core team will begin to draft a document for the trustee meeting
+The core team will start to look into Kickstart venues
+The core team will confirm timing/van details for the sorting session this weekend, and send the information out.
```
Comment (PeterJCLaw) [inline 2018/2018-09-05.md]: This one was on me.

File: 2018/2018-09-05.md (hunk 1 above)
Comment (BKiddo) [reply 2018/2018-09-05.md]: Changed and submitted for review

**Pull request merged**